packages/ethereum/tasks/faucet.ts

```diff
@@ -12,10 +11,7 @@ const send = (signer, txparams) =>
     console.log(`transactionHash: ${transactionHash}`)
   })
 
-const nativeAddress = async (hoprAddress) => {
-  const nodePeerPubkey = await convertPubKeyFromB58String(hoprAddress)
-  return utils.computeAddress(nodePeerPubkey.marshal())
-}
+const nativeAddress = async (hoprAddress: string) => convertPubKeyFromB58String(hoprAddress).toAddress().toHex()
```

Comment:
```suggestion
const nativeAddress = async (hoprAddress: string) => PublicKey.fromPeerId(PeerId.createFromB58String(hoprAddress)).toAddress().toHex()
```

Then we can delete these utils which are otherwise unused.

Reply:
afaik these function are used in some chat bots and somewhere in the deploy pipeline, hence I'd postpone this step

Reply:
can you grep the codebase for them? if there are no results, it's safe to remove.